packages/ai/rollup.config.js

```diff
@@ -57,12 +58,14 @@ const browserBuilds = [
     plugins: [
       ...buildPlugins,
       replace({
+        ...getEnvironmentReplacements('browser'),
         ...generateBuildTargetReplaceConfig('esm', 2020),
-        __PACKAGE_VERSION__: pkg.version
+        '__PACKAGE_VERSION__': pkg.version
       }),
       emitModulePackageFile()
     ],
-    external: id => deps.some(dep => id === dep || id.startsWith(`${dep}/`))
+    external: id =>
```

Comment:
FYI "external" doesn't really do anything in most of our package rollup files as you have to add plugins `@rollup/plugin-node-resolve` and `@rollup/plugin-commonjs` in order to bundle dependencies, which I got very frustrated at while trying to figure out why removing/modifying `external` didn't do anything.

Reply:
I added this line to quiet warnings for a missing `ws` dependency, which is an external dependency on the native Node APIs.